Scholars Portal Journals.js

```diff
@@ -35,23 +35,26 @@
 	***** END LICENSE BLOCK *****
 */
 
-function detectWeb(doc, url) { 
-	var myArticles = doc.getElementById('my-articles');
-	if (myArticles) {
-  		Zotero.monitorDOMChanges(myArticles.parentElement, {attributes: true, attributeFilter: ["style"]});
-	}
-	// see if saved list is toggled open
-	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
+function detectWeb(doc, url) {
+	if (url.indexOf("/my-articles") != -1) {
```

Comment:
We prefer -- and now strictly enforce -- `string.includes()` for testing literal matches -- please fix throughout